client/package.json

```diff
@@ -83,10 +83,12 @@
     "prop-types": "^15.5.10",
     "react": "^15.5.4",
     "react-ace": "^4.3.0",
+    "react-color": "^2.13.1",
     "react-dnd": "^2.4.0",
     "react-dnd-html5-backend": "^2.4.1",
     "react-dom": "^15.5.4",
     "react-dropzone": "^3.13.2",
+    "react-fabricjs": "^0.1.6",
```

Comment:
Also I think we should make these libs aync and only load on submissions page, similar to what we did to the Intl: https://github.com/Coursemology/coursemology2/blob/master/client/app/index.js#L37